app/main.py

```diff
@@ -2,5 +2,30 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    def wrapper(*args, **kwargs) -> None:
```

Comment:
The `wrapper` function is defined to return `None`, but it should return the same type as the decorated function. Consider changing the return type annotation to match the return type of the decorated function.